actionkit/api/order.py

```diff
@@ -1,8 +1,45 @@
 from actionkit.api.base import ActionKitAPI
 
-
 class AKOrderAPI(ActionKitAPI):
 
+    def get_order_detail(self, order_id):
+        """
+            Get order and billing info
+        """
+        result = self.client.get(
+            '%s/rest/v1/order/%s' % (
+                self.base_url, order_id))
+        rv = {'res': result}
+
+        if result.status_code == 200:
+            json = result.json()
+            rv.update(json)
+            user_detail = json.get('user_detail')
+            user_detail_res = self.client.get('%s%s' % (self.base_url, user_detail))
+            if user_detail_res.status_code == 200:
+                rv['user_detail'] = user_detail_res.json()
```

Comment:
If there are any important keys in user_detail that are frequently used, I would suggest copying those to the base `rv` dict -- that way the consumer of the api doesn't need to test inner dicts to get the data they need for this request (which is presumably run because we want data of both)d